index.src.html

```diff
@@ -822,10 +830,10 @@ <h4 id="clear-dom">
      
       1.  An origin's WebSQL databases [[!WEBDATABASE]].
       2.  An origin's filesystems [[!file-system-api]]
-      3.  ChannelID
-      4.  Plugin data (e.g. <a href="https://wiki.mozilla.org/NPAPI:ClearSiteData">NPP_ClearSiteData</a>)
-      5.  Appcache.
-      6.  ISSUE: Moar?
+      3.  Plugin data (e.g. <a href="https://wiki.mozilla.org/NPAPI:ClearSiteData">NPP_ClearSiteData</a>),
+          except Flash [[!FLASH]] LSOs (see step #7 in [[#clear-cookies]]).
```

Comment:
Yes, we did argue about this one and ended up in a state when both of us acknowledged the other position :) Which means that I'm happy to move it to "storage" as well.

My argument was

a) the nickname "flash cookies" (which I admit is not a convincing one)

b) the fact that LSOs are not origin-scoped like other "storage" data. We're deleting "cookies" for eTLD+1, and it's easy to say what it means to delete LSOs for eTLD+1. We delete "storage" for the origin, so now we'll also have to clarify what does it mean to clear an LSO for an origin.  Specifically, that "www.a.example.org" will be able to erase LSOs from "www.b.example.org", because LSOs are scoped to "example.org".